node/pkg/telemetry/prom_remote_write/scrape.go

```diff
@@ -0,0 +1,108 @@
+package promremotew
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/golang/snappy"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+type PromTelemetryInfo struct {
+	PromRemoteURL string
+	StatusPort    uint16
+	NodeName      string
+}
+
+func scrapeLocalMetrics(ctx context.Context, metricsPort uint16) ([]byte, error) {
+	// The idea is to grab all the metrics from localhost:6060/metrics,
+	// and then send them to Grafana.
+	metricsURL := fmt.Sprintf("http://localhost:%d/metrics", metricsPort)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, metricsURL, nil)
```

Comment:
is there a better was to get this data from the prometheus registry? Could you simply `.Gather()` the metrics?

Reply:
Possibly.  Depends on the definition of "better".  This would require rewriting all of format.go

Reply:
Changed.